src/Data/Uniques/belt.lua

```diff
@@ -400,34 +400,42 @@ Variant: Pre 3.11.0 (Strength and Intelligence)
 {tags:jewellery_resistance}+(30-40)% to Cold Resistance
 Chill nearby Enemies when you Focus, causing 30% reduced Action Speed
 {variant:1,2,3,4,5,6,7,8,9,10}Focus has (15-25)% increased Cooldown Recovery Rate
-{variant:11,12,13,14,15,16,17,18,19,20}Focus has (30-50)% increased Cooldown Recovery Rate
+{variant:11,12,13,14,15,16,17,18,19}Focus has (30-50)% increased Cooldown Recovery Rate
 (50-70)% increased Damage with Hits and Ailments against Chilled Enemies
-{variant:1,11}{crafted}{tags:life}Regenerate 2% of Life per second during any Flask Effect
-{variant:2,12}{crafted}{tags:jewellery_resistance}+(8-15)% to Fire and Chaos Resistances
-{variant:3,13}{crafted}{tags:jewellery_resistance}+(8-15)% to Cold and Chaos Resistances
-{variant:4,14}{crafted}{tags:jewellery_resistance}+(8-15)% to Lightning and Chaos Resistances
-{variant:5,15}{crafted}{tags:jewellery_attribute}+(6-17) to Strength and Dexterity
-{variant:6,16}{crafted}{tags:jewellery_attribute}+(6-17) to Dexterity and Intelligence
-{variant:7,17}{crafted}{tags:jewellery_attribute}+(6-17) to Strength and Intelligence
-{variant:8,18}{crafted}{tags:speed}(7-12)% increased Trap Throwing Speed
-{variant:9,19}{crafted}{tags:jewellery_defense}Regenerate 120 Energy Shield per second while a Rare or Unique Enemy is Nearby
-{variant:10,20}{crafted}Your Critical Strike Chance is Lucky while Focused
+{variant:1}{crafted}{tags:life}Regenerate 2% of Life per second during any Flask Effect
+{variant:2}{crafted}{tags:chaos,jewellery_resistance}+(8-15)% to Fire and Chaos Resistances
+{variant:3}{crafted}{tags:chaos,jewellery_resistance}+(8-15)% to Cold and Chaos Resistances
+{variant:4}{crafted}{tags:chaos,jewellery_resistance}+(8-15)% to Lightning and Chaos Resistances
+{variant:5}{crafted}{tags:jewellery_attribute}+(6-17) to Strength and Dexterity
+{variant:6}{crafted}{tags:jewellery_attribute}+(6-17) to Dexterity and Intelligence
+{variant:7}{crafted}{tags:jewellery_attribute}+(6-17) to Strength and Intelligence
+{variant:8}{crafted}(7-12)% increased Trap Throwing Speed
+{variant:9}{crafted}{tags:jewellery_defense}Regenerate 120 Energy Shield per second while a Rare or Unique Enemy is Nearby
+{variant:10,19}{crafted}Your Critical Strike Chance is Lucky while Focused
+{variant:11}{tags:chaos,jewellery_resistance}+(16-20)% to Fire and Chaos Resistances
+{variant:12}{tags:chaos,jewellery_resistance}+(16-20)% to Cold and Chaos Resistances
+{variant:13}{tags:chaos,jewellery_resistance}+(16-20)% to Lightning and Chaos Resistances
+{variant:14}{tags:jewellery_attribute}+(31-35) to Strength and Dexterity
+{variant:15}{tags:jewellery_attribute}+(31-35) to Dexterity and Intelligence
+{variant:16}{tags:jewellery_attribute}+(31-35) to Strength and Intelligence
+{variant:17}(14-16)% increased Trap Throwing Speed
+{variant:18}{tags:jewellery_defense}Regenerate 200 Energy Shield per second while a Rare or Unique Enemy is Nearby
+{variant:19}Focus has (5-8)% increased Cooldown Recovery Rate
```

Comment:
These should have `{crafted}` in front of them.

Reply:
```lua
{variant:11}{crafted}{tags:chaos,jewellery_resistance}+(16-20)% to Fire and Chaos Resistances
{variant:12}{crafted}{tags:chaos,jewellery_resistance}+(16-20)% to Cold and Chaos Resistances
{variant:13}{crafted}{tags:chaos,jewellery_resistance}+(16-20)% to Lightning and Chaos Resistances
{variant:14}{crafted}{tags:jewellery_attribute}+(31-35) to Strength and Dexterity
{variant:15}{crafted}{tags:jewellery_attribute}+(31-35) to Dexterity and Intelligence
{variant:16}{crafted}{tags:jewellery_attribute}+(31-35) to Strength and Intelligence
{variant:17}{crafted}(14-16)% increased Trap Throwing Speed
{variant:18}{crafted}{tags:jewellery_defense}Regenerate 200 Energy Shield per second while a Rare or Unique Enemy is Nearby
{variant:19}{crafted}Focus has (5-8)% increased Cooldown Recovery Rate
```